public/app/features/dashboard/services/performanceObserverUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { getScenePerformanceTracker, writePerformanceLog, type ScenePerformanceObserver } from '@grafana/scenes';
```

Comment:
I think this entire file can be removed as `performanceUtils` seems to contain it